### PR TITLE
fix: fixing all css mess

### PR DESCRIPTION
**Pull request by mariyasada**
fixing all the CSS code. changing width of buttons

### CODE REVIEW

File: stylesheets/product-list/productList.css [hunk 1]
```diff
@@ -95,7 +104,7 @@
 .btns-in-flex {
     display: flex;
     align-items: center;
-    justify-content: center;
+    /* justify-content: center; */
```
Comment (fahd23) [inline stylesheets/product-list/productList.css]: Remove Commented codes

File: stylesheets/profile/user-profile.css [hunk 2]
```diff
@@ -1,60 +1,63 @@
...
+.btn-tabs:hover {
     background-color: var(--medium-teal);
 }
-.details-container
-{
-    width:100%;
+
+.details-container {
+    width: 100%;
     /* box-shadow: 2px 2px 4px var(--light-stone); */
```
Comment (fahd23) [inline stylesheets/profile/user-profile.css]: same here

**Review by fahd23 — APPROVED** [review]
Good to merge

**Review by atharvabhanage02 — APPROVED** [review]
Looks Good to Merge

File: stylesheets/cart/cart.css [hunk 3]
```diff
@@ -1,90 +1,104 @@
...
 .card-qty-flex,
-.item-price-detail
-{
+.item-price-detail {
     display: flex;
-    gap:0.3rem;
+    gap: 0.4rem;
     align-items: center;
-    justify-content: center;
+    /* justify-content: center; */
```
Comment (shaksham08) [inline stylesheets/cart/cart.css]: Remove this comments if not necessary

**Review by shaksham08 — APPROVED** [review]
Looks good to me, added one comment

**Pull request merged**